recipes/rebound/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "REBOUND" %}
+{% set version = "3.4.0" %}
+{% set sha256 = "d070bfd7a8c48a28776b4bae5ba4dcf22d81e7776cd518dc3a3479972104d5bc" %}
+
+package:
+  name: rebound
+  version: "3.4.0"
+
+source:
+  fn: rebound-3.4.0.tar.gz
+  url: https://pypi.python.org/packages/34/c0/a74c26085adb72ec4672d09f08cfabb68e917f9e68a3c71293ffa08fa82c/rebound-3.4.0.tar.gz

```

Comment:
Please change to the newer style url -- see this recipe for an example: https://github.com/conda-forge/staged-recipes/pull/3011/files